pkg/cosign/cosign_test.go

```diff
@@ -0,0 +1,234 @@
+package cosign
+
+import (
+	"context"
+	_ "embed"
+	"fmt"
+	"io/ioutil"
```

Comment:
Just like the bot says, this is causing CI validation to fail.

Reply:
Yup, didn't check that, fixed.